src/tests/SensorTest.cpp

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) Facebook, Inc. and its affiliates.
+// This source code is licensed under the MIT license found in the
+// LICENSE file in the root directory of this source tree.
+#include <Corrade/Utility/Directory.h>
+#include <gtest/gtest.h>
+
+#include "esp/scene/SceneManager.h"
+#include "esp/scene/SceneNode.h"
+#include "esp/sensor/Sensor.h"
+#include "esp/sensor/SensorFactory.h"
+
+#include "configure.h"
+
+using namespace esp::sensor;
+using namespace esp::scene;
+
+// TODO: Add tests for different Sensors
+
+TEST(SensorTest, SensorFactory) {
```

Comment:
This is a newly added sensor test, which is really great! Thank you.

Unfortunately, you are following an old test template which is using google test.
We are migrating the test to use the `Cr::TestSuite::Tester`. See the `SimTest.cpp` you have modified.
Would you change it?